Broadcast-Ext/README.md

```diff
@@ -0,0 +1,34 @@
+Broadcast Extension sample app
+===========================
+
+The Broadcast Extension app shows how to implement the iOS Broadcast Upload extension
+using the OpenTok iOS SDK. The publisher's audio track is turned off by default. 
+You can enable the audio track by setting the `audioTrack` parameter in the `doPublish:` method
+in OTBroadcastExtHelper.m. When you enable the publisher's audio track, be sure to turn on
+the microphone in the Broadcast UI dialog box. Otherwise the publisher will fail because
```

Comment:
what happens if you don't check the mic button because you don't want to stream your microphone?
What means the publisher will fail?

Reply:
Routed sessions always expect both video and audio packets when you enable audioTrack and videoTrack, but when mic is not enabled there won't be any audio samples and eventually the publisher gets timeout. One workaround is to send silent audio samples when the mic is disabled but it's tricky to detect when the mic is enabled/disabled (AFAIK there are no notifications for that). IMO adding silent audio logic may make make the sample complex, but we can add that in the next version.

Reply:
We can explain this in the README too, something like: 
```As our platform expects audio and video data, if you don't allow mic access there will be no audio and then It will fail```